common.lic

```diff
@@ -155,6 +155,12 @@ module DRC
         put message
         timer = Time.now
         next
+      when /^You must be standing/, /^You should stand up first/, /^You can't do that while (sitting|kneeling|lying)/
```

Comment:
If I'm reading this code right 
https://github.com/rpherbig/dr-scripts/blob/7953d9c04f406c6e8da141c97c4e898de9b7924a/coordinator.lic#L459
then there is one more case: `'You should be sitting up'`

Reply:
If I'm reading this code right 
https://github.com/rpherbig/dr-scripts/blob/45ae6be4d505615899eadfd30b54ddfb339b8fe1/stabbity.lic#L140
then there is one more case: `You'll need to stand up first`

Reply:
If I'm reading this code right 
https://github.com/rpherbig/dr-scripts/blob/acc89c9a453fcb5afe06e4fa36e31e8b7b8bd4dd/boggle_blast.lic#L97
then there is one more case: `You really should be standing to play`

Reply:
`Boggle_blast.lic` is a dead script at this point. Should we remove all the old HE games, the ones from The Spider?

Reply:
Admittedly git will keep the history, but we may want to move them into an archive folder, just in case.